Makefile

```diff
@@ -206,16 +234,20 @@ operator-image:
 	rm -rf ${JUJUD_STAGING_DIR}
 
 push-operator-image: operator-image
+## push-operator-image: Push up the build operator image via docker
 	docker push ${OPERATOR_IMAGE_PATH}
 
 microk8s-operator-update: operator-image
+## microk8s-operator-update: Save and update the operator image for use with microk8s
```

Comment:
build, save and update...

may want to add that `... Save and update the operator image in registry`